pubsub/driver/driver.go

```diff
@@ -78,6 +78,11 @@ type Message struct {
 	AfterSend func(asFunc func(interface{}) bool) error
 }
 
+// Implement batcher item for byte sized based batching
```

Comment:
Nit: "ByteSize estimates the size in bytes of the message for the purpose of restricting batch sizes."